developer-docs/providers/metaschema.md

```diff
@@ -0,0 +1,789 @@
+# Pulumi Package Metaschema
```

Comment:
Is this auto-generated or hand-authored?

Reply:
This is generated by the `jsonschema2md` tool that is part of this PR: https://github.com/pulumi/pulumi/pull/7952/files#diff-98fe9f26fbfe318ee4b2589b4c08d4d693cea4f721b10cc3be93b659e73b47dc